xyz

```diff
@@ -90,29 +138,25 @@ while (($# > 0)) ; do
       exit
       ;;
     -v|--version)
-      node -p "require('$dir/package.json').version"
+      node -p 'require("xyz/package.json").version'
       exit
       ;;
-    -b|--branch)          branch="$1"           ; shift ;;
-    -e|--edit)            edit=true             ;;
-    -i|--increment)       increment="$1"        ; shift ;;
-    -m|--message)         message_template="$1" ; shift ;;
-       --publish-command) publish_command="$1"  ; shift ;;
-    -r|--repo)            repo="$1"             ; shift ;;
-    -s|--script)          scripts+=("$1")       ; shift ;;
-    -t|--tag)             tag_template="$1"     ; shift ;;
-       --dry-run)         dry_run=true          ;;
+    -b|--branch)            branch="$1"           ; shift ;;
+    -e|--edit)              edit=true             ;;
+    -i|--increment)         increment="$1"        ; shift ;;
+    -m|--message)           message_template="$1" ; shift ;;
+       --prerelease-label)  prerelease_label="$1" ; shift ;;
+       --publish-command)   publish_command="$1"  ; shift ;;
+    -r|--repo)              repo="$1"             ; shift ;;
+    -s|--script)            scripts+=("$1")       ; shift ;;
+    -t|--tag)               tag_template="$1"     ; shift ;;
+       --dry-run)           dry_run=true          ;;
     *)
       echo "Unrecognized option $option" >&2
       exit 1
   esac
 done
 
-case "$increment" in
-  major|minor|patch|premajor|preminor|prepatch|prerelease) ;;
-  *) echo "Invalid --increment" >&2 ; exit 1 ;;
-esac
-
```

Comment:
The `--increment` value is validated in `inc`, so this paragraph is no longer necessary.